backend/src/server/routes/v3/secret-router.ts

```diff
@@ -300,6 +300,11 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
         secretPath: z.string().trim().default("/").transform(removeTrailingSlash).describe(RAW_SECRETS.GET.secretPath),
         version: z.coerce.number().optional().describe(RAW_SECRETS.GET.version),
         type: z.nativeEnum(SecretType).default(SecretType.Shared).describe(RAW_SECRETS.GET.type),
+        expandSecretReferences: z
+          .enum(["true", "false"])
+          .default("false")
+          .transform((value) => value === "true")
+          .describe(RAW_SECRETS.GET.expand),
```

Comment:
would it be more appropriate if this was singular?
since we're dealing with individual secrets here...

Reply:
I actually think it makes sense

A secret can contain multiple references, it's not **a** reference, it may be multiple references to other secrets

I.e: TEST-SECRET: ${dev.secret-1}<>${dev.secret-2}.${dev.other-secret}

Reply:
ahhh right, it can be viewed that way as well.. I was thinking of "secret reference" referring to the collective of references but that makes sense